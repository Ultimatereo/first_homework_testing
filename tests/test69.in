5
-170754 -9338578 6658634 -8324316 -9506219 4442195
1982108 6117808 -6060036 2416725 -7817211 -7002076
2624721 9312650 8353762 -6478272 458549 -6628110
436483 4286706 -348492 -352442 -5135740 -557642
-2895496 2112819 211292 3371972 2862655 -75546
