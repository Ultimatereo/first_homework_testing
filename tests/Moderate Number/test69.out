-29.692689
5.899262
-24.104800
-33.155605
6.420000